frontend/packages/ceph-storage-plugin/src/constants/resources.ts

```diff
@@ -0,0 +1,19 @@
+import { FirehoseResource } from '@console/internal/components/utils/index';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
+import { CephClusterModel } from '../models';
+import { CEPH_STORAGE_NAMESPACE } from './index';
+
+const cephClusterResource: FirehoseResource = {
```

Comment:
should be synced with https://github.com/openshift/console/pull/2433/ - ideally reuse the same definition